civihr_employee_portal/css/custom.css

```diff
@@ -230,6 +230,29 @@ span.appraisals-employee-legend {
   width: calc(100% - 30px);
 }
 
+#reportPivotTable .report-fields-selection table li {
+  margin-bottom: 5px;
+  width: 100%;
+}
+
+#reportPivotTable .report-fields-selection table li span.pvtAttr,
+#reportPivotTable .report-field-rows table li span.pvtAttr {
+  height: auto;
+  line-height: 1.2em;
+  padding: 8px 20px 8px 15px;
+  padding-right: 20px;
+  position: relative;
+  white-space: normal;
+  width: 100%;
```

Comment:
Since you have `width` defined here:
```css
#reportPivotTable .report-fields-selection table li {
  margin-bottom: 5px;
  width: 100%;
}
```
Here you can inherit the width (please check):
`width: inherit;`

Reply:
For this particular one, when I try it, it sets the width to 150px, not 100%. The table is 100%, but the li is not a direct descendant of the table.